### PR TITLE
CloudServices 43-preview release update

### PR DESCRIPTION
**Pull request by Sandido**
preview.1 update to ensure the AssemblyInfo file is updated as expected.

### CODE REVIEW

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("42.0.0.0")]
-[assembly: AssemblyFileVersion("42.0.0.0")]
+[assembly: AssemblyVersion("43.0.0.0-preview.1")]
```
Comment (markcowl) [inline sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs:10]: Generally we don't do this witl assembly file versions.  This should be 43.0.0.1

**Review by markcowl — CHANGES_REQUESTED** [review]

File: sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (Sandido) [reply sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs]: @markcowl , edited as you wanted, please re-review.

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**